### PR TITLE
Add new sentence in langs

### PR DESCRIPTION
**Pull request by olsza**
After the positive merge of PR #65, I add a new sentence to be translated in other translation languages, so that the translation maintainers would be easier to translate without adding it to the file;)

CC (the guardians of languages):
- [x] DE: @dkaufmann96 
- [x] EN: @mattstauffer (leader) 
- [x] ES: @wilsenhc
- [x] FR: @aurelien31
- [x] ID: @akunbeben
- [x] IT: @Lukasss93
- [x] JA: @kitar
- [x] LT: @LukasKuc
- [x] PL: @olsza (my)
- [x] pt_BR: @danrcastro
- [x] RU: @Hkr32
- [x] UK: @vmartuniyk
- [x] VI: @nguyenyou
- [x] zh_CN: @w3yyb

### CODE REVIEW

File: resources/lang/it.json [hunk 1]
```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Per aggiornare, segui le istruzioni nella documentazione o usa :link-laravelshifts per aggiornare automaticamente.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Aggiornare <em>almeno</em> ad una versione che supporta gli aggiornamenti di sicurezza <strong>il più presto possibile!",
     "Update to the latest major or LTS release.": "Aggiorna all'ultima versione major o LTS.",
-    "Version": "Versione"
+    "Version": "Versione",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (Lukasss93) [inline resources/lang/it.json:45]: "You can also find a list of all of the security advisories for Laravel here: :link": "Inoltre potrai trovare una lista di tutti gli avvisi di sicurezza per Laravel qui: :link"

Comment (olsza) [reply resources/lang/it.json]: Thanks

File: resources/lang/pt_BR.json [hunk 2]
```diff
@@ -40,5 +40,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Para atualizar, siga as instruções na documentação ou use :link-laravelshifts para atualizar automaticamente.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Atualize <em>pelo menos</em> para uma versão que recebe correções de segurança <strong>o mais rápido possível!</strong>",
     "Update to the latest major or LTS release.": "Atualize para a versão mais recente ou LTS",
-    "Version": "Versão"
+    "Version": "Versão",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (danrcastro) [inline resources/lang/pt_BR.json:44]: "You can also find a list of all of the security advisories for Laravel here: :link": "Você também pode encontrar uma lista de todos os alertas de segurança para o Laravel aqui: :link"

File: resources/lang/ru.json [hunk 3]
```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Для обновления следуйте инструкциям в документации или используйте :link-laravelshifts для автоматического обновления.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Обновите <em>как минимум</em> до версии с сохранением безопасности <strong>как можно скорее!</strong>",
     "Update to the latest major or LTS release.": "Обновите до последнего основного релиза или релиза с долгосрочной поддержкой(LTS).",
-    "Version": "Версия"
+    "Version": "Версия",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (Hkr32) [inline resources/lang/ru.json:45]: "You can also find a list of all of the security advisories for Laravel here: :link": "Вы также можете найти список всех рекомендаций по безопасности для Laravel здесь: :link"

File: resources/lang/pt_BR.json (hunk 2 above)
Comment (olsza) [reply resources/lang/pt_BR.json]: Thanks

File: resources/lang/ru.json (hunk 3 above)
Comment (olsza) [reply resources/lang/ru.json]: Thanks

File: resources/lang/vi.json [hunk 4]
```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Để nâng cấp, hãy làm theo hướng dẫn trong tài liệu hoặc sử dụng :link-laravelshifts để nâng cấp tự động.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Cập nhật <em>tối thiểu</em> sang phiên bản được bảo trì lỗi bảo mật <strong>sớm nhất có thể!</strong>",
     "Update to the latest major or LTS release.": "Cập nhật lên bản phát hành chính gần nhất hoặc LTS mới nhất.",
-    "Version": "Phiên bản"
+    "Version": "Phiên bản",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (nguyenyou) [inline resources/lang/vi.json:45]:
```suggestion
    "You can also find a list of all of the security advisories for Laravel here: :link": "Bạn cũng có thể tìm thấy danh sách tất cả các cố vấn bảo mật cho Laravel tại đây: :link"
```

**Review by akunbeben — COMMENTED** [review]
very nice

File: resources/lang/vi.json (hunk 4 above)
Comment (olsza) [reply resources/lang/vi.json]: Thanks

File: resources/lang/fr.json [hunk 5]
```diff
@@ -42,5 +42,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Pour mettre à niveau, suivez les instructions de la documentation ou utilisez :link-laravelshifts pour mettre à jour automatiquement.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Mettre à jour <em>au moins</em> vers une version sécurisée <strong>dès que possible !</strong>",
     "Update to the latest major or LTS release.": "Mettre à jour vers la dernière version majeure ou LTS.",
-    "Version": "Version"
+    "Version": "Version",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (aurelien31) [inline resources/lang/fr.json:46]:
``` suggestion
    "You can also find a list of all of the security advisories for Laravel here: :link": "Vous pouvez également trouver une liste de tous les bulletins de sécurité pour Laravel ici : :link"
```

**Review by aurelien31 — CHANGES_REQUESTED** [review]

File: resources/lang/es.json [hunk 6]
```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Para actualizar, sigue las instrucciones en la documentación o utiliza :link-laravelshifts para actualizar automáticamente.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "¡Actualice <em>al menos</em> a una versión con mantenimiento de seguridad <strong>lo más pronto que pueda!</strong>",
     "Update to the latest major or LTS release.": "Actualice a la versión principal más reciente o la versión LTS.",
-    "Version": "Versión"
+    "Version": "Versión",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (wilsenhc) [inline resources/lang/es.json:45]: "You can also find a list of all of the security advisories for Laravel here: :link": "También puedes encontrar una lista de todos los avisos de seguridad para Laravel aquí: :link"

Comment (wilsenhc) [reply resources/lang/es.json:45]: Commit [3044220](https://github.com/wilsenhc/laravelversions/commit/3044220957ee01619d6292f5a168e633f95ce267) in my fork already adds  this change

File: resources/lang/fr.json (hunk 5 above)
Comment (olsza) [reply resources/lang/fr.json:46]: Thanks

File: resources/lang/es.json (hunk 6 above)
Comment (olsza) [reply resources/lang/es.json]: Thanks, I also put your change into my PR, but you want it too can throw your PR ;)

**Comment by olsza** [comment]
I think @mattstauffer that this PR can already be uploaded. In a week, more than half of the language translations of a new sentence were collected. :)

File: resources/lang/ja.json [hunk 7]
```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "アップグレードするには、ドキュメントのインストラクションに従うか :link-laravelshifts の自動アップグレードを利用してください。",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "<em>少なくとも</em>セキュリティ修正が行われているバージョンまで<strong>できる限り早く</strong>アップデートしてください！",
     "Update to the latest major or LTS release.": "メジャーまたはLTSリリースにアップデートしてください。",
-    "Version": "バージョン"
+    "Version": "バージョン",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (kitar) [inline resources/lang/ja.json:45]: "You can also find a list of all of the security advisories for Laravel here: :link": "Laravelのすべてのセキュリティアドバイザリはこちらでもご覧いただけます: :link"

**Review by kitar — CHANGES_REQUESTED** [review]

File: resources/lang/ja.json (hunk 7 above)
Comment (olsza) [reply resources/lang/ja.json]: Thanks

File: resources/lang/de.json [hunk 8]
```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Um zu upgraden, befolgen Sie die Anleitung in der Dokumentation oder verwenden Sie :link-laravelshifts um automatisch zu upgraden.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Updaten Sie <strong>sobald wie möglich</strong> auf eine Version, welche <em>zumindest</em> in Bezug auf Security-Fixes gewartet wird!",
     "Update to the latest major or LTS release.": "Updaten Sie auf die neueste Hauptversion oder das Langzeit-Support Release.",
-    "Version": "Version"
+    "Version": "Version",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (dkaufmann96) [inline resources/lang/de.json:45]:
```suggestion
    "You can also find a list of all of the security advisories for Laravel here: :link": "Hier finden Sie eine Auflistung aller Security-Berater für Laravel: :link"
```

Comment (dkaufmann96) [reply resources/lang/de.json:45]: `"You can also find a list of all of the security advisories for Laravel here: :link": "Hier finden Sie eine Auflistung aller Security-Berater für Laravel: :link"`

**Review by anubhav217 — APPROVED** [review]

File: resources/lang/de.json (hunk 8 above)
Comment (olsza) [reply resources/lang/de.json:45]: Thanks

**Review by anubhav217 — COMMENTED** [review]
Langs added successfully!

**Review by vmartuniyk — COMMENTED** [review]
"You can also find a list of all of the security advisories for Laravel here: :link": "Ви також можете знайти список усіх рекомендацій щодо безпеки для Laravel тут:: link"

File: resources/lang/uk.json [hunk 9]
```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Для оновлення слідуйте інструкції з документації або використовуйте :link-laravelshifts для автоматичного оновлення.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Оновіть <em> принаймні </em> до захищеної версії <strong> якомога швидше! </strong>",
     "Update to the latest major or LTS release.": "Оновіть до останнього основного випуску або випуску LTS.",
-    "Version": "Версія"
+    "Version": "Версія",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (vmartuniyk) [inline resources/lang/uk.json:45]: "You can also find a list of all of the security advisories for Laravel here: :link": "Ви також можете знайти список усіх рекомендацій щодо безпеки для Laravel тут:: link"

**Review by vmartuniyk — COMMENTED** [review]
"You can also find a list of all of the security advisories for Laravel here: :link": "Ви також можете знайти список усіх рекомендацій щодо безпеки для Laravel тут:: link"

File: resources/lang/id.json [hunk 10]
```diff
@@ -41,5 +41,6 @@
     "To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Untuk meningkatkan, ikuti perintah-perintah yang ada di dokumentasi atau gunakan :link-laravelshifts untuk peningkatan otomatis.",
     "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Perbarui <em>setidaknya</em> ke versi keamanan yang terkelola <strong>secepatnya!</strong>",
     "Update to the latest major or LTS release.": "Perbarui ke rilis LTS major terakhir.",
-    "Version": "Versi"
+    "Version": "Versi",
+    "You can also find a list of all of the security advisories for Laravel here: :link": "You can also find a list of all of the security advisories for Laravel here: :link"
```
Comment (akunbeben) [inline resources/lang/id.json:45]: > "You can also find a list of all of the security advisories for Laravel here: :link": "Anda juga dapat menemukan daftar dari semua saran keamanan untuk Laravel disini: :link"

**Review by akunbeben — CHANGES_REQUESTED** [review]
> "You can also find a list of all of the security advisories for Laravel here: :link": "Anda juga dapat menemukan daftar dari semua saran keamanan untuk Laravel disini: :link"

**Comment by wilsenhc** [comment]
Only zh_CN @w3yyb and LT @LukasKuc translations missing now, right?

**Review by w3yyb — COMMENTED** [review]
yes

**Review by w3yyb — APPROVED** [review]

**Review by anubhav217 — APPROVED** [review]
Build running so far. Checked all node modules

File: resources/lang/uk.json (hunk 9 above)
Comment (olsza) [reply resources/lang/uk.json]: Thanks...
 and sorry for being so late i was on vacation for 3 weeks

File: resources/lang/id.json (hunk 10 above)
Comment (olsza) [reply resources/lang/id.json]: thanks

**Comment by olsza** [comment]
I apologize to everyone that I did not give the sign, I was on vacation for 3 weeks without internet ;)

**Comment by olsza** [comment]
If users over the weekend @w3yyb and @LukasKuc not translate this sentence, I have a proposal to make @matt merge, and Translations these two languages, i.e. zh_CN and LT will be in separate PR

**Comment by olsza** [comment]
> If users over the weekend @w3yyb and @LukasKuc not translate this sentence, I have a proposal to make @matt merge, and Translations these two languages, i.e. zh_CN and LT will be in separate PR

@mattstauffer This PR has been open for a long time, and the authors of These two translations zh_CN and LT, don't let me know, maybe you will merge these translations? It is a pity for the efforts of the other 12 authors of translations;)

**Comment by wilsenhc** [comment]
Maybe have the zh_CN and LT translations open in a new issue for first time contributors?

**Comment by LukasKuc** [comment]
> > If users over the weekend @w3yyb and @LukasKuc not translate this sentence, I have a proposal to make @matt merge, and Translations these two languages, i.e. zh_CN and LT will be in separate PR
> 
> @mattstauffer This PR has been open for a long time, and the authors of These two translations zh_CN and LT, don't let me know, maybe you will merge these translations? It is a pity for the efforts of the other 12 authors of translations;)

I have done LT, but it is still pending.

**Comment by olsza** [comment]
> > > If users over the weekend @w3yyb and @LukasKuc not translate this sentence, I have a proposal to make @matt merge, and Translations these two languages, i.e. zh_CN and LT will be in separate PR
> > 
> > 
> > @mattstauffer This PR has been open for a long time, and the authors of These two translations zh_CN and LT, don't let me know, maybe you will merge these translations? It is a pity for the efforts of the other 12 authors of translations;)
> 
> I have done LT, but it is still pending.

hey @LukasKuc, will you send the link, where are they? because I can't see :(

**Review by w3yyb — CHANGES_REQUESTED** [review]
ok

**Comment by olsza** [comment]
hey @w3yyb, maybe you can give one sentence to translate the above-mentioned ones, here in the comment, and I'll do a commit;)
ps. @LukasKuc, maybe you can also give a translation here ??

**Comment by LukasKuc** [comment]
> hey @w3yyb, maybe you can give one sentence to translate the above-mentioned ones, here in the comment, and I'll do a commit;) ps. @LukasKuc, maybe you can also give a translation here ??

image

it was here, I thought you can see that because it says pending. :)

`"You can also find a list of all of the security advisories for Laravel here: :link": "Taip pat visas Laravel saugumo rekomendacijas galite rasti čia: :link"`

**Review by w3yyb — APPROVED** [review]
"You can also find a list of all of the security advisories for Laravel here: :link": "您还可以在此处找到Laravel的所有安全公告列表: :link"

**Comment by olsza** [comment]
> > hey @w3yyb, maybe you can give one sentence to translate the above-mentioned ones, here in the comment, and I'll do a commit;) ps. @LukasKuc, maybe you can also give a translation here ??
> 
> image
> 
> it was here, I thought you can see that because it says pending. :)
> 
> `"You can also find a list of all of the security advisories for Laravel here: :link": "Taip pat visas Laravel saugumo rekomendacijas galite rasti čia: :link"`

Thank you

**Comment by olsza** [comment]
> ```
> "You can also find a list of all of the security advisories for Laravel here: :link": "您还可以在此处找到Laravel的所有安全公告列表: :link"
> ```


Thank you

**Comment by olsza** [comment]
yopi, @mattstauffer we have FULL translations, you can implement :) :D

**Comment by mattstauffer** [comment]
@olsza awesome, thank you!!!

**Comment by mattstauffer** [comment]
Thank you to all the contributors here!

**Pull request merged**